TPL/Omnistudio/DocumentGeneration/DocGen_Card.json

```diff
@@ -0,0 +1 @@
\":\"Current Window\",\"layoutType\":\"lightning\",\"flyoutType\":\"OmniScripts\",\"openFlyoutIn\":\"Modal\",\"channelName\":\"close_modal\",\"osName\":\"docGenerationSample/fndSingleDocxLwc/English\",\"flyoutLwc\":\"doc-generation-sample-fnd-single-docx-lwc-english\",\"hasExtraParams\":true,\"flyoutParams\":{\"ContextId\":\"{recordId}\"}},\"key\":\"1686585713337-oak8v23og\",\"label\":\"Action\",\"draggable\":false,\"isOpen\":true,\"actionIndex\":0,\"contextId\":\"\\\\{recordId}\",\"reRenderFlyout\":true}],\"showSpinner\":\"false\",\"flyoutChannel\":\"close_modal\",\"flyoutDetails\":{\"openFlyoutIn\":\"Modal\"},\"displayAsButton\":true,\"buttonVariant\":\"brand\",\"iconOnly\":false,\"hideActionIcon\":true,\"reRenderFlyout\":true},\"type\":\"element\",\"styleObject\":{\"sizeClass\":\"slds-size_12-of-12\"},\"elementLabel\":\"Start\",\"userUpdatedElementLabel\":true,\"key\":\"element_element_block_0_0_action_1_0\",\"parentElementKey\":\"element_block_0_0\"}],\"elementLabel\":\"Block-1\"}]}},\"childCards\":[],\"actions\":[],\"omniscripts\":[],\"documents\":[]}],\"dataSource\":{\"type\":null,\"value\":{},\"orderBy\":{},\"contextVariables\":[]},\"title\":\"DocGen_Card\",\"enableLwc\":true,\"isFlex\":true,\"theme\":\"slds\",\"selectableMode\":\"Multi\",\"xmlObject\":{\"targetConfigs\":\"PHRhcmdldENvbmZpZyB0YXJnZXRzPSJsaWdodG5pbmdfX0FwcFBhZ2UiPgogICAgICAgICAgICAgICAgICAgICAgPHByb3BlcnR5IG5hbWU9ImRlYnVnIiB0eXBlPSJCb29sZWFuIi8+CiAgICAgICAgICAgICAgICAgICAgICA8cHJvcGVydHkgbmFtZT0icmVjb3JkSWQiIHR5cGU9IlN0cmluZyIvPgogICAgICAgICAgICAgICAgICA8L3RhcmdldENvbmZpZz4KICAgICAgICAgICAgICAgICAgPHRhcmdldENvbmZpZyB0YXJnZXRzPSJsaWdodG5pbmdfX1JlY29yZFBhZ2UiPgogICAgICAgICAgICAgICAgICAgICAgPHByb3BlcnR5IG5hbWU9ImRlYnVnIiB0eXBlPSJCb29sZWFuIi8+CiAgICAgICAgICAgICAgICAgIDwvdGFyZ2V0Q29uZmlnPg==\",\"targets\":{\"target\":[\"lightning__RecordPage\",\"lightning__AppPage\",\"lightning__HomePage\"]}},\"xmlJson\":[{\"@attributes\":{\"targets\":\"lightning__AppPage\"},\"property\":[{\"@attributes\":{\"name\":\"debug\",\"type\":\"Boolean\"}},{\"@attributes\":{\"name\":\"recordId\",\"type\":\"String\"}}]},{\"@attributes\":{\"targets\":\"lightning__RecordPage\"},\"property\":[{\"@attributes\":{\"name\":\"debug\",\"type\":\"Boolean\"}}]}],\"lwc\":{\"DeveloperName\":\"cfDocGen_Card_1_CGI\",\"Id\":\"0Rb8G000001DHzaSAG\",\"MasterLabel\":\"cfDocGen_Card_1_CGI\",\"NamespacePrefix\":\"c\",\"ManageableState\":\"unmanaged\"}}","OmniUiCardType":"Parent","Id":"0ko8G000000Gn7aQAC","VlocityDataPackIsIncluded":true,"StylingConfiguration":"","UniqueName":""}],"VlocityDataPackType":"FlexCard","VlocityDataPackKey":"d58b2d8fb2c619dba4fa5d68f25a113f"},"VlocityDataPackAllRelationships":{},"DataPackAttachmentSize":7784,"DataPackAttachmentParentId":"0nc8G00000002OhQAI","DataPackAttachmentId":"00P8G0000082nqKUAQ","ActivationStatus":"Ready"}],"dataPackId":"0nc8G00000002OhQAI","alreadyExportedKeys":[],"name":"DocGen_Card","description":"Document Generation Flex Card for flyout modal","version":1,"Status":"Complete"}
```

Comment:
The core change is the implementation of a flyout modal so that recalculation of required pages for the Minister's Certificate is recalculated every time the process is started. This fixes the previous issue is that the process could be left mid-interview and would not recheck the available costs when they went back to the in-progress interview.